backend/src/services/sync/landmarkSyncService.test.ts

```
@@ -0,0 +1,134 @@
+ */
+
+import { describe, it, expect, vi } from 'vitest';
+
+vi.mock('../../db/index.js', () => ({
+  pool: { query: vi.fn(), connect: vi.fn() },
+}));
+
+import { bindingsToLandmarks, fetchMonuments } from './landmarkSyncService.js';
+import type { SparqlBinding } from './wikidataUtils.js';
+
+const ENTITY = 'http://www.wikidata.org/entity/';
```

ESLint

[error] Using http protocol is insecure. Use https instead.

(sonarjs/no-clear-text-protocols)